src/main/kotlin/app/revanced/patches/music/premium/backgroundplay/BackgroundPlayPatch.kt

```diff
@@ -0,0 +1,25 @@
+package app.revanced.patches.music.premium.backgroundplay
+
+import app.revanced.extensions.exception
+import app.revanced.patcher.data.BytecodeContext
+import app.revanced.patcher.extensions.InstructionExtensions.addInstructions
+import app.revanced.patcher.patch.BytecodePatch
+import app.revanced.patcher.patch.annotation.CompatiblePackage
+import app.revanced.patcher.patch.annotation.Patch
+import app.revanced.patches.music.premium.backgroundplay.fingerprints.BackgroundPlaybackDisableFingerprint
+
+
+@Patch(
+    name = "Background play",
+    description = "Enables playing music in the background.",
+    compatiblePackages = [CompatiblePackage("com.google.android.apps.youtube.music")]
+)
+object BackgroundPlayPatch : BytecodePatch(setOf(BackgroundPlaybackDisableFingerprint)) {
```

Comment:
❓ : Missing unused suppression
> `BackgroundPlayPatch` is unused and should be suppressed.

```suggestion
@Suppress("unused")
object BackgroundPlayPatch : BytecodePatch(setOf(BackgroundPlaybackDisableFingerprint)) {
```